tests/device_fixtures.py

```diff
@@ -124,6 +124,7 @@
 HUBS_SMART = {"H100", "KH100"}
 SENSORS_SMART = {"T310", "T315", "T300", "T100", "T110", "S200B", "S200D"}
 THERMOSTATS_SMART = {"KE100"}
+VACUUMS_SMART = {"RV30"}
```

Comment:
Use the prefix, or the full name? There exist different RV30 versions in different markets, so it might be necessary to make this more strict later.